package.json

```diff
@@ -137,6 +137,7 @@
     "style-loader": "^1.0.0",
     "terminate": "^2.1.0",
     "to-string-loader": "^1.1.5",
+    "web-ext": "^5.0.0",
```

Comment:
I think rather than adding this to our package json, we should just add a line to the instructions to tell users to install `web-ext` globally.

`yarn global add web-ext`

Reply:
That was what I had started the instructions with, but was using NPM not yarn. 

I'll make these changes and resubmit.

Reply:
Yeah, the more important part is just removing it from the dependency list, it doesn't need to be there for someone to install globally.

Reply:
should be resolved now after reverting my last commit.